src/js/stores/ui.js

```diff
@@ -298,7 +298,9 @@ define(function (require, exports, module) {
                 right = (columns !== undefined && singleColumn) ? 0 :
                     this._iconBarWidth;
 
-            if (columns === undefined || columns === this._columnCount) {
+            if (singleColumn) {
+                right += this._panelWidth;
```

Comment:
coulda just added `singleColumn` to the list of disjuncts in the existing if condition?